tests/unit/test-util.js

```diff
@@ -0,0 +1,75 @@
+const sinon = require('sinon');
+const { expect } = require('chai');
+const appRoot = require('app-root-path');
+const camelCase = require('camelcase');
+const _ = require('lodash');
+const config = require('config');
+
+sinon.stub(config, 'get').returns({ oracledb: {} });
+const conn = appRoot.require('api/v1/db/oracledb/connection');
+const { openapi } = appRoot.require('utils/load-openapi');
+const { fakeBaseUrl } = require('./test-data');
```

Comment:
Base on our [style guide](https://github.com/osu-mist/style-guides/blob/master/JavaScript.md#require-statement-order), requires a local file should be put in front of the `appRoot` require.